tests/test_cmdline.py

```diff
@@ -9,8 +10,19 @@
 
 
 def test_commandline_basic(tmpdir):
-    """Simple command line usage should work"""
-    subprocess.check_call([sys.executable, VIRTUALENV_SCRIPT, str(tmpdir.join("venv"))])
+    """Simple command line usage should work and files should be generated"""
+    venv_dir = str(tmpdir.join("venv"))
+    venv_bin_dir = os.path.join(venv_dir, "Scripts" if sys.platform == "win32" else "bin")
```

Comment:
this seems a bit brittle (easy to break), maybe instead re-use the path locations logic from https://github.com/pypa/virtualenv/blob/0571b2ef5a612fa1c6a61c205b631de47aa1fc7f/src/virtualenv.py#L1025 by calling to that function to find out the bin directory

Reply:
That function will also make the directory, but I will add a flag to skip this `mkdir` line when running the tests.

https://github.com/pypa/virtualenv/blob/0571b2ef5a612fa1c6a61c205b631de47aa1fc7f/src/virtualenv.py#L1035